packages/shared-core/src/sdk/documents/users.ts

```diff
@@ -129,3 +130,30 @@ export function containsUserID(value: string | undefined): boolean {
   }
   return value.includes(`${DocumentType.USER}${SEPARATOR}`)
 }
+
+export function getUserGroups(user: User, groups?: UserGroup[]) {
+  return (
+    groups?.filter(group => group.users?.find(u => u._id === user._id)) || []
+  )
+}
+
+export function getUserAppGroups(
+  appId: string,
+  user: User,
+  groups?: UserGroup[]
+) {
+  const prodAppId = getProdAppID(appId)
+  const userGroups = getUserGroups(user, groups)
+  return userGroups.filter(group =>
+    Object.keys(group.roles || {}).find(app => app === prodAppId)
+  )
+}
+
+export function userAppAccessList(user: User, groups?: UserGroup[]) {
```

Comment:
Couldn't this be used to populate the user app list in the portal?

Reply:
Yeap - sorry I missed the app list in the portal - thanks for catching that!

Reply:
Updated this to a similar experience in the user side panel.

![image](https://github.com/user-attachments/assets/ed28e6cf-a53f-4fb8-bc68-311b1c92e381)